scripts/pylint_extensions.py

```diff
@@ -1252,12 +1253,15 @@ def process_module(self, node):
             # shebang expression at the start of a bash script which
```

Comment:
Should this be: "Check that the comment **doesn't** start with a space"?

Reply:
Here we are checking if the comment starts with space and is not a shebang expression. I think this is fine too.

Reply:
I don't understand -- in the other comments, you state properties that the comment _should_ have (e.g. "check that the comment ends with the proper punctuation"). I'm a bit confused about whether "having a space character at the start and is not a shebang expression" is a desirable or undesirable property.

Actually, looking through the file, it seems like the comments are inconsistent. Could we just reformat all of them to specify desirable properties of comments? E.g.

- "Comments must end with the proper punctuation."
- "Comments may include a lowercase character at the beginning only if they start with version info or a data type."

etc. That will help with standardization and make the communication clearer.

Thanks!

Reply:
Done.